src-tauri/tests/fixtures/bundler/import_jsx/input/index.jsx

```diff
@@ -0,0 +1,15 @@
+import getMessage from "./utils";
```

Comment:
```suggestion
import getMessage from "./MessageInner";
```

We commonly name the file of a React component exactly by the name of that component,